react-node-mongodb/server/server.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Created by Syed Afzal
+ */
+require('./config/config');
+
+const express = require('express');
+const path = require('path');
+const cookieParser = require('cookie-parser');
+const bodyParser = require('body-parser');
+const cors = require('cors');
+const db = require('./db');
+
+
+
+const app = express();
+
+//connection from db here
+db.connect();
```

Comment:
If you use the previous snippet in `db/index.js` don't forget to pass `app` to the function

Reply:
Done